src/transformers/utils/import_utils.py

```diff
@@ -345,6 +345,12 @@ def is_accelerate_available(min_version: str = ACCELERATE_MIN_VERSION):
     return _accelerate_available and version.parse(_accelerate_version) >= version.parse(min_version)
 
 
+def is_torch_accelerator_available():
+    import torch
+
+    return is_torch_available() and hasattr(torch, "accelerator")
```

Comment:
we should `is_torch_available` first before `import torch` 🙏

Reply:
thx, done, and also fixed the similar issue in `is_torch_deterministic`